e2e/utils/create-project-utils.ts

```diff
@@ -76,10 +76,12 @@ export function newProject({
   name = uniq('proj'),
   packageManager = getSelectedPackageManager(),
   packages,
+  workspaces = false,
 }: {
   name?: string;
   packageManager?: 'npm' | 'yarn' | 'pnpm' | 'bun';
   readonly packages?: Array<NxPackage>;
+  workspaces?: boolean;
```

Comment:
I think we should just allow setting the preset. So, the usage would be `newProject({ preset: 'ts' })`.

The `--workspaces` flag is not needed for the `ts` preset. It's actually ignored with that preset. It would only be needed when using another preset that supports the new TS setup behind the flag (right now, only React). We don't generate any of the React presets in the e2e tests, so it doesn't seem we would need to use it. We can still add it to support using other presets that do need it.